neurots/generate/orientations.py

```diff
@@ -16,15 +16,33 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import inspect
+import warnings
+from copy import deepcopy
 
+import neurom as nm
 import numpy as np
+from scipy.optimize import curve_fit
+from scipy.special import expit
 
 from neurots.morphmath import rotation
 from neurots.morphmath import sample
 from neurots.morphmath.utils import normalize_vectors
+from neurots.utils import PIA_DIRECTION
 from neurots.utils import NeuroTSError
 
 _TWOPI = 2.0 * np.pi
+fit_3d_angles_bounds = {
+    "double_step": ([0, 0.1, -np.pi, 0.1], [np.pi, 10, 0, 10]),
+    "step": ([-np.pi, 0.1], [np.pi, 10]),
+}
+fit_3d_angles_params = {
+    "with_apical": {"basal_dendrite": {"form": "step", "bounds": fit_3d_angles_bounds["step"]}},
+    "without_apical": {"basal_dendrite": {"form": "flat", "bounds": []}},
+}
+_3d_angles_mapping = {
+    "apical_constraint": "apical_3d_angles",
+    "pia_constraint": "pia_3d_angles",
+}
```

Comment:
I would use caps for these global variable to show that they should not be changed by the user.